source/utils/layout.js

```diff
@@ -1,4 +1,53 @@
 // @flow
+import { pickBy, isEmpty } from 'lodash';
+
+type GridItemProps = {
+  alignSelf: string,
+  column: string | number,
+  columnStart: string | number,
+  columnEnd: string | number,
+  gridArea: string,
+  justifySelf: string,
+  placeSelf: string,
+  row: string | number,
+  rowEnd: string | number,
+  rowStart: string | number,
+};
+
+export const assembleInlineGridItem = (gridItemProps: GridItemProps) => {
```

Comment:
@DominikGuzei 

I originally wrote `GridItem` as a class based component, but it wasn't using the constructor and it had no state. It's only method, besides `render`, was `_assembleInlineGridItem`. Because of these reason, I figured it would be better to change `GridItem` to a stateless functional component. From there, I could either make make `assembleInlineGridItem` a function expression within the component, or pull it into into utils/layout.js.

Considering this function isn't going to be used by another component and there is a duplication of the `GridItemProps` type definition, I'll pull it back into GridItem.js. Good catch, good catch!